docs/samples/Microsoft.ML.Samples/Dynamic/SimpleDataViewImplementation.cs

```diff
@@ -0,0 +1,254 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.ML;
+using Microsoft.ML.Data;
+
+namespace Samples.Dynamic
+{
+    /// <summary>
+    /// The <see cref="IDataView"/> interface is the central concept of "data" in ML.NET. While many conveniences exist
+    /// to create pre-baked implementations, it is also useful to know how to create one completely from scratch. We also
+    /// take this opportunity to illustrate and motivate the basic principles of how the IDataView system is architected,
+    /// since people interested in implementing <see cref="IDataView"/> need at least some knowledge of those principles.
+    /// </summary>
```

Comment:
we're using simple, non-xml comments since the sample displays as is.

Reply:
+1

---
In reply to: [274765737](https://github.com/dotnet/machinelearning/pull/3302#discussion_r274765737) [](ancestors = 274765737)

Reply:
But why? Maybe we've had wildly difference experiences, but I've generally observed that people don't *read* samples as an intellectual exercise from beginning to end, they *might* read a little, but immediately fall to *running* them and see what the heck is going on. At least, such has been my observation. It's definitely true for me.

But, you're the expert, but I don't know that I like this idea of making the documentation non-browsable if someone actually tries to run the sample. Perhaps that's what you're doing, but I might prefer to have some understanding of why it's actually a good idea.